gateway/1.8/modules/ROOT/pages/_partials/task-config-readiness-liveness.adoc

```diff
@@ -0,0 +1,187 @@
+//tag::config-readiness-liveness-page-body[]
+
+Flex Gateway supports two methods for ensuring reliability and efficiency.
+
+* Readiness probes test whether a gateway instance is configured correctly and ready for incoming traffic. This ensures that end-users are not affected by unstable or partially started services. For information, refer to <<configure-a-readiness-probe>>.
+
+* Liveness probes test whether a gateway instance is operational. This minimizes downtime due to crashes or other failures. For information, refer to <<configure-a-liveness-probe>>.
+
+[[configure-a-readiness-probe]]
+== Configure a Readiness Probe
+
+To test whether a Flex Gateway instance is ready for incoming traffic, you can use the following CLI command:
+
+[source,ssh]
+----
+flexctl probe --check=readiness
+----
+
+If the Flex Gateway instance is ready to handle incoming traffic, the command returns an exit code of `0`. Otherwise the command returns an exit code of `1`.
+
+When Flex Gateway is started, its readiness state is `false`. The state changes to `true` after the following conditions are met:
+
+* API instances are deployed (at least one API must be configured)
+* Policies are initialized
+
+The readiness state is not impacted when you apply configuration changes to a running gateway instance. This enables error-free scaling.
+
+Refer to the following for information about configuring readiness probes for specific environments:
+
+[tabs]
+====
+Kubernetes and OpenShift::
++
+By default, Flex Gateway running on Kubernetes includes a readiness probe configured in the Helm chart. The readiness probe automatically runs `flexctl probe --check=readiness` at an interval of 10 seconds.
```

Comment:
I'm not sure if you want to add this flags too, but they are configured by default:

```suggestion
By default, Flex Gateway running on Kubernetes includes a readiness probe configured in the Helm chart. The readiness probe automatically runs `flexctl probe --check=readiness --allow-api-errors --allow-envoy-errors --allow-policy-errors` at an interval of 10 seconds.
```